internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/fastclient/AvroStoreClientEndToEndTest.java

```diff
@@ -228,12 +228,13 @@ protected void runTest(
     }
   }
 
-  @Test(dataProvider = "FastClient-Four-Boolean-A-Number-Store-Metadata-Fetch-Mode", timeOut = TIME_OUT)
+  @Test(dataProvider = "FastClient-Five-Boolean-A-Number-Store-Metadata-Fetch-Mode", timeOut = TIME_OUT)
```

Comment:
We use this to test the server-side stats collected by gRPC enabled server with fast clients. This increases the total amount of time that integration tests take to run by a significant amount D: Would separating these tests be a good idea in this case?

Reply:
Thanks for flagging this, but I think it's fine... this class is part of `IntegrationTestsI`, and also extended by a couple of other classes which are part of `IntegrationTestsG`. Based on your latest CI run, these two tasks took 20m and 15m respectively, which is less than the longest running task, so it should not increase the total time the test suite takes.